google-cloud-storage/test/google/cloud/storage/project_test.rb

```diff
@@ -1022,7 +1082,19 @@ def list_buckets_gapi count = 2, token = nil
     )
   end
 
+  def list_deleted_buckets_gapi count = 2, token = nil
+    buckets = count.times.map { Google::Apis::StorageV1::Bucket.from_json random_deleted_bucket_hash.to_json }
+    Google::Apis::StorageV1::Buckets.new(
+      kind: "storage#buckets", items: buckets, next_page_token: token
+    )
+  end
+
+  def restored_bucket_gapi name
+    Google::Apis::StorageV1::Bucket.from_json random_bucket_hash(name: name).to_json
+  end
+
+
   def object_retention_param enable_object_retention
     enable_object_retention ? Google::Apis::StorageV1::Bucket::ObjectRetention.new(mode: "Enabled") : nil
   end
-end
```

Comment:
You probably want to revert this change back and add the newline character back.

Reply:
did not understand your concerns here